### PR TITLE
Cleanup MathML elements and attribute pages

### PR DESCRIPTION
**Pull request by fred-wang**
#### Summary

- Add a note that we focus on subset relevant for the web.
- remove mention of the obsolete XLink (strictly speaking not part of MathML)
- remove mention that mstyle/math support all attributes. This no longer correct.
- move deprecated `background`, `color`, `fontfamily`, `fontsize`, `fontstyle`, `fontweight` attributes into their own row.
- remove attributes that are not implemented in any browser (and a fortiori not in MathML Core)
- remove mention of "presentation" and "content" MathML, which are not really relevant for web developers.

#### Motivation

More up-to-date doc, focusing on what's relevant for browsers.

#### Supporting details

N/A

#### Related issues

N/A

#### Metadata

This PR…
-->
- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by fred-wang** [comment]
cc @teoli2003

File: files/en-us/web/mathml/element/index.md [hunk 1]
```diff
@@ -7,13 +7,11 @@ tags:
 ---
 {{MathMLRef}}
 
-This is an alphabetical list of MathML **presentation** elements.
+This is an alphabetical list of MathML elements. All of them implement the {{domxref("MathMLElement")}} class.
 
-The term **presentation markup** is used to describe the layout structure of mathematical notation whereas **content markup** provides the underlying mathematical meaning and is not supposed to be rendered by the MathML parser (see {{ bug(276028) }}). If you want to learn more about content markup you should consider [Chapter 4](https://www.w3.org/TR/MathML3/chapter4.html) in the [MathML 3 specification](https://www.w3.org/TR/MathML3/).
+> **Note:** As explained of the main [MathML](/en-US/docs/Web/MathML) page, documentation is focused on a subset of the language that is relevant for browsers. See the [MathML Full](https://w3c.github.io/mathml/) specification if you want details about other MathML elements.
```
Comment (teoli2003) [inline files/en-us/web/mathml/element/index.md:12]:
```suggestion
> **Note:** As explained on the main [MathML](/en-US/docs/Web/MathML) page, documentation is focused on a subset of the language that is relevant for browsers. See the [MathML Full](https://w3c.github.io/mathml/) specification if you want details about other MathML elements.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by fred-wang** [comment]
@teoli2003 can you please merge? (I don't think I have github permission on this repo to merge or ask review request BTW)

**Comment by teoli2003** [comment]
I was waiting for the tests to finish.

**Pull request merged**